glade_forms/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: glade_forms
 description: A universal way to define form validators with support of translations.
-version: 2.2.0
+version: 2.3.0
```

Comment:
Dont increase version here yet. Just mention changes in changelog's "DEV" section

Reply:
Oh, wasn't familiar, how things are done here, version bump rolled back